railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_5_2.rb.tt

```diff
@@ -14,6 +14,11 @@
 # Existing cookies will be converted on read then written with the new scheme.
 # Rails.application.config.action_dispatch.use_authenticated_cookie_encryption = true
```

Comment:
While we're here we may as well say that this also breaks cookie backwards compatibility with 5.1.

We could also consider unifying the 2 options into one. Or just read `use_authenticated_cookie_encryption` as embed the expiry.

Reply:
Yeah. I'll use `use_authenticated_cookie_encryption` for both configs.